html/excercise_three.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Excersie Three</title>
+  </head>
+  <body>
+    <h2>Organize your monthly tasks</h2>
+
+    <p>Create and control your onthly tasks to never miss one. You can mark the tasks as completed. If you <br>are reaching end of month and some task is not complete, we will sen you a reminder! Each<br> month the task will automatically get back to uncompleted.</p>
+
+    <a href="">Create Task</a><br>
+    <a href="">All users</a><br><br>
+
+    <ul>
+      <br><label><li>Pay co-living rent <input type="checkbox"></li></label><br>
```

Comment:
don't think you need <br> here at the beginning, also `<li> `should be the first thing after `<ul>` something like this:

```
<ul>
      <li>Something</li>
      <li>Something</li>
      <li>Something</li>
</ul>
```
this automatically put the spaces on the list.